components/sponsor/NftSubmissionItem.vue

```diff
@@ -29,19 +31,19 @@ const { t } = useI18n({ useScope: 'global' });
             NFT #{{ submission.nftId }}
           </RuiChip>
           <RuiChip
-            v-if="nftMetadata?.tier"
+            v-if="isDefined(submission.tierId)"
             size="sm"
             variant="outlined"
-            :class="getTierClasses(nftMetadata?.tier)"
+            :class="getTierClasses(findTierById(submission.tierId)?.key)"
           >
             <span class="mr-1">
-              {{ getTierMedal(nftMetadata.tier) }}
+              {{ getTierMedal(findTierById(submission.tierId)?.key) }}
             </span>
-            <span class="uppercase font-medium mr-1">{{ t('sponsor.submit_name.tier_info', { tier: nftMetadata.tier }) }}</span>
-            <span v-if="nftMetadata?.releaseName">
+            <span class="uppercase font-medium mr-1">{{ t('sponsor.submit_name.tier_info', { tier: findTierById(submission.tierId)?.label }) }}</span>
```

Comment:
This is the tier badge right? can you extract it to a component?

So that we pass the tierId as a parameter and make findTierById into the computed instead of calling the same function 3 times in the template?